examples/bedrock-agent-fastapi/app/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM public.ecr.aws/docker/library/python:3.12.0-slim
+COPY --from=public.ecr.aws/awsguru/aws-lambda-adapter:0.8.1 /lambda-adapter /opt/extensions/lambda-adapter
+ENV PORT=8000
```

Comment:
Add `AWS_LWA_READINESS_CHECK_PROTOCOL=tcp` to use TCP connect for healthcheck

```Dockerfile
ENV PORT=8000 AWS_LWA_READINESS_CHECK_PROTOCOL=tcp
```

Reply:
fixed